src/05-derivatives/02-common-data-types.md

```diff
@@ -71,16 +71,16 @@ The `space` entity may take any value in [Image-Based Coordinate Systems][coords
-| SpatialReference | RECOMMENDED if the derivative is aligned to a standard template listed in [Standard template identifiers][templates]. REQUIRED otherwise. | [string][] or [object][] | For images with a single reference, the value MUST be a single string. For images with multiple references, such as surface and volume references, a JSON object MUST be used. See examples below. |
+| **Key name**     | **Requirement level**                                                                                                                     | **Data type**            | **Description**                                                                                                                                                                                                                            |
+| ---------------- | ----------------------------------------------------------------------------------------------------------------------------------------- | ------------------------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| SpatialReference | RECOMMENDED if the derivative is aligned to a standard template listed in [Standard template identifiers][templates]. REQUIRED otherwise. | [string][] or [object][] | For images with a single reference, the value MUST be a [BIDS URI][]. Using a single string is [DEPRECATED][]. For images with multiple references, such as surface and volume references, a JSON object MUST be used. See examples below. |
 
 ### SpatialReference key allowed values
 
-| **Value**   | **Description**                                                                                                               |
-| ----------- | ----------------------------------------------------------------------------------------------------------------------------- |
-| `orig`      | A (potentially unique) per-image space. Useful for describing the source of transforms from an input image to a target space. |
-| URI or path | This can be used to point to a specific file. Paths are written relative to the root of the derivative dataset.               |
+| **Value**    | **Description**                                                                                                                        |
+| ------------ | -------------------------------------------------------------------------------------------------------------------------------------- |
+| `"orig"`     | A (potentially unique) per-image space. Useful for describing the source of transforms from an input image to a target space.          |
+| [BIDS URI][] | This can be used to point to a specific file. URIs and paths written relative to the root of the derivative dataset are [DEPRECATED][] |
```

Comment:
```suggestion
| **Value** | **Description**                                                                                                                        |
| --------- | -------------------------------------------------------------------------------------------------------------------------------------- |
| `"orig"`  | A (potentially unique) per-image space. Useful for describing the source of transforms from an input image to a target space.          |
| URI       | This can be used to point to a specific file. Relative paths are [DEPRECATED][] in favor of [BIDS URIs][BIDS URI].          |
```

Didn't fix the table alignment for description.

Reply:
Thanks, I adjusted this, will push soon.